avalon/pipeline.py

```diff
@@ -192,10 +192,14 @@ class Loader(list):
     order = 0
     is_multiple_contexts_compatible = False
 
+    options = []
+
     def __init__(self, context):
-        self.fname = self.filepath_from_context(context)
+        self.fname = Loader.filepath_from_context(context)
 
-    def filepath_from_context(self, context):
+    @classmethod
+    def filepath_from_context(cls, context):
```

Comment:
Why this became classmethod?

Reply:
My bad, `staticmethod` would be better. The method is not changing state and could be useful event without initialized object.

Reply:
But there are some Loaders that have overriden this method and static definition in default will break them... Keep it as object mehod.